### ISSUE TITLE
On Country Map, Cross-Filtering is not operational as a source

### ISSUE THREAD

**Issue by xavier-GitHub76** [apache/superset#29623]
Hello,

Country Map can be  filtered by the cross-filtering but it is not operational as a source because a click do zoom on map

Best regards

**Comment by rusackas** [comment]
Yep. This is not a bug, really, but a lack of design/implementation for that plugin. Any interest in taking this on? We might close this as an Issue, but I'm happy to move it to an "Ideas" discussion if you want to figure out how to solve it or try to find help with it.

**Comment by xavier-GitHub76** [comment]
I agree that you move this issue to a discussion

**Comment by imad-hl** [comment]
Any updates on this subject ?
